player/lua/stats.lua

```diff
@@ -80,7 +80,7 @@ local o = {
     ass_it0 = "{\\i0}",
     -- Without ASS
     no_ass_nl = "\n",
-    no_ass_indent = "\t",
+    no_ass_indent = "    ",
```

Comment:
How to pass such value with `script-opts`

Reply:
`--script-opts-append=stats-no_ass_indent=foo`

Reply:
No I was asking how to pass `space` in script-opts

Reply:
And how could it know how many spaces if append a comment?
```
# mpv.conf
script-opts-append=stats-no_ass_indent=<space><space><space> # This is a comment.
```

Reply:
`script-opts-append='stats-no_ass_indent=  '`

Reply:
`--script-opts-append=stats-no_ass_indent="    "`

Reply:
" " get included in the option value that way.

Reply:
> " " get included in the option value that way.

```
Audio: AAC (Advanced Audio Coding) [LC]  AO: pipewire
    Device: Autoselect device  AO Volume: 100%
    Channels: 2  Format: floatp
    Sample Rate: 44100 Hz
    Bitrate: 128 kbps
    Filters: @dynnorm: dynaudnorm (disabled) [g=15 r=1]
```

Works here?

Reply:
Yeah it works in the CLI but he was talking about mpv.conf.